platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/bls/protocol/ProtocolOutput.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.bls.protocol;
+
+/** An interface representing the output object of a {@link BlsProtocol} */
+public interface ProtocolOutput {
```

Comment:
Consider renaming to `BlsProtocolOutput`

Reply:
I've refactored the code, and this interface no longer exists